app/src/androidTest/java/org/mozilla/fenix/ui/util/Strings.kt

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.ui.util
+
+const val STRING_ONBOARDING_ACCOUNT_SIGN_IN_HEADER = "Sync Firefox between devices"
```

Comment:
Let's add a comment for this file to indicate its purpose is for holding commonly used strings in the UI tests.

Reply:
I was also originally thinking we could keep the constants within the same test file if we didn't use them in multiple files.

Reply:
Yes, that will be ideal but we are using the same strings in multiple files.